### DISCUSSION TITLE
Postgres driver set everything to lowercase

### DISCUSSION THREAD

**Discussion by renrenproject** [dbeaver/dbeaver#20333] [Q&A]
### Description

The table name & column name whatever I typed will modified to lower case automatically. 
As we know postgres has case sensitive, if you select a column name doesn't match to the actual column it will occur "[Perhaps you meant to reference the column ...]"
<img width="598" alt="image" src="https://github.com/dbeaver/dbeaver/assets/33349734/23e2428d-dfbd-42cf-a00d-56c0ae144b90">
BTW, the pgAdmin doesn't have this issue, and the MS SQL Server driver also doesn't have this issue.

### DBeaver Version

23.1.0

### Operating System

Windows 11

### Database and driver

PostgreSQL 15
Postgres driver 2.5.0

**Comment by LonwoLonwo** [comment]
Hello @wwy12321 

You can turn off this behavior with this setting:

![2023-06-15 14_13_38-Window](https://github.com/dbeaver/dbeaver/assets/45152336/d3400643-2b74-42cf-8332-a78da16cd708)

**Comment by renrenproject** [comment]
Thanks @LonwoLonwo ,
Another issue is text in SQL editor also change to lowercase.
<img width="491" alt="image" src="https://github.com/dbeaver/dbeaver/assets/33349734/38443b08-4da3-420c-975f-91d121df4496">
And all the columns inside that table also changed to lowercase.

**Comment by renrenproject** [comment]
Hi @LonwoLonwo , any results?

**Comment by E1izabeth** [comment]
@wwy12321 
To use CamelCase naming you need to put the identifier to quotes. That's how PostgreSQL works.

**Comment by renrenproject** [comment]
Thanks @E1izabeth
